cdap-docs/reference-manual/source/glossary.rst

```diff
@@ -130,5 +130,9 @@ Glossary
       independent of another instance of the same entity in a different namespace. The
       primary motivation for namespaces in CDAP is to achieve application and data
       isolation.
+
+   Master Services
+      YARN containers started by the ``cdap-master`` process. This includes the containers that perform logging,
```

Comment:
CDAP system services that are run in YARN containers like Transaction Service, Dataset Executor, Log Saver, Metrics Processor, etc.